.github/workflows/ci.yml

```diff
@@ -45,6 +45,7 @@ jobs:
             - name: Run the django tests
               run: |
                   make minio
+                  docker network create grand-challengeorg_workstations
```

Comment:
The latest release of docker compose only creates the network if they are attached to one of the running containers, so we need to manually create this here.